### PR TITLE
Migrate resolver to async-await syntax.

### PR DESCRIPTION
**Pull request by cbracken**

### CODE REVIEW

**Comment by kevmoo** [comment]
oops...

**Comment by cbracken** [comment]
Fixed. +1 for running assertions on :)

**Comment by kevmoo** [comment]
See https://github.com/dart-lang/coverage/pull/85

**Comment by cbracken** [comment]
Even better than #85.

File: lib/src/resolver.dart [hunk 1]
```diff
@@ -84,22 +84,11 @@ class Loader {
...
-          .catchError((e) {
-        failed.add(uri);
-        c.complete(null);
-      });
-      return c.future;
+  Future<List<String>> load(String path) async {
+    try {
+      return new File(path).readAsLines();
+    } catch (_) {
+      return null;
```
Comment (kevmoo) [inline lib/src/resolver.dart]: do we still care about populating `failed`?

**Comment by kevmoo** [comment]
Love it. Just one nit.

File: lib/src/resolver.dart (hunk 1 above)
Comment (cbracken) [inline lib/src/resolver.dart]: Yep. Done.

**Comment by kevmoo** [comment]
ship it

**Pull request merged**